internal/services/compute/managed_disk_resource.go

```diff
@@ -215,6 +215,17 @@ func resourceManagedDisk() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			"hyper_v_generation_version": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
whilst this comes from the image, since this is the disk resource we should make users either specify this, or use ignore_changes on it:

```suggestion
```

Reply:
Computed removed